lib/utils/getServerlessConfigFile.js

```diff
@@ -19,12 +25,12 @@ const getServerlessConfigFilePath = (srvcPath) => {
     yaml: fileExists(yamlPath),
     js: fileExists(jsPath),
   }).then(exists => {
-    if (exists.json) {
-      return jsonPath;
+    if (exists.yaml) {
+      return yamlPath;
     } else if (exists.yml) {
       return ymlPath;
-    } else if (exists.yaml) {
-      return yamlPath;
+    } else if (exists.json) {
+      return jsonPath;
```

Comment:
It's technically a breaking change, (now `serverless.yml` will be favored over `sererless.json`, but I guess that can be assumed as a bug fix (?)

Reply:
yeah, I this a bugfix since there were internal inconsistencies in priority